src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs

```diff
@@ -269,6 +270,39 @@ protected override IEnumerable<IMenuBarItem> AllMenuBarItems()
             .WithIcon(ApplicationIcons.Simulation)
             .WithShortcut(Keys.Control | Keys.Alt | Keys.Shift | Keys.S);
 
+         yield return CreateMenuButton.WithCaption(AppConstants.Captions.AddIndividual)
+            .WithId(MenuBarItemIds.NewIndividual)
+            .WithDescription(ToolTips.SimulationRibbon.CreateSimulation)
+            .WithCommand<AddNewIndividualCommand>(_container)
+            .WithDescription(ToolTips.ModelingRibbon.CreateIndividual)
+            .WithIcon(ApplicationIcons.Individual);
+
+         yield return CreateMenuButton.WithCaption(AppConstants.Captions.Module)
+            .WithId(MenuBarItemIds.NewExtensionModule)
+            .WithCommand<NewModuleWithBuildingBlocksUICommand>(_container)
+            .WithDescription(ToolTips.ModelingRibbon.CreateExtensionModule)
+            .WithIcon(ApplicationIcons.Module);
```

Comment:
Could you consider adding keyboard shortcuts? I think there's like Ctrl-Alt-shift-S for simulation